aspnet/web-pages/readme/beta3.md

```diff
@@ -501,4 +499,4 @@ For more information about WebMatrix Beta 3, see the following websites:
 
 ---
 
-© 2010 Microsoft Corporation. All Rights Reserved. [Terms of Use](https://msdn.microsoft.cos/cc300389.aspx).
+© 2010 Microsoft Corporation. All Rights Reserved. [Terms of Use](https://www.microsoft.com/en-us/servicesagreement).
```

Comment:
There's already a Terms of Use link on the site, so just delete this entire line. Thanks!